pkg/cri/resource-manager/requests.go

```diff
@@ -461,16 +466,77 @@ func (m *resmgr) UpdateContainer(ctx context.Context, method string, request int
 	return &criapi.UpdateContainerResourcesResponse{}, nil
 }
 
+type ContainerStats struct {
+	CPU    float64 `containermetrics:"cpu"`
+	Memory float64 `containermetrics:"wss"`
+}
+
+type NumaMetrics struct {
+	Total       float64 `containermetrics:"numa_total"`
+	File        float64 `containermetrics:"numa_file"`
+	Anon        float64 `containermetrics:"numa_anon"`
+	Unevictable float64 `containermetrics:"numa_unevictable"`
+}
+
```

Comment:
It would be nice to see a proposed 'top-level/final' per-container data structure with all data we intend to produce/collect ATM. I think that structure should
  - have the capability to include both total and per-core CPU usage (something we can get from the `cpuacct` controller)
  - have the capability to include both total and per-node NUMA stats
  - make it possible to differentiate data not being present from metrics data being 0 (struct pointers)

I also wonder if the sliding window, EWMA, etc. implementation should be visible as a mandatory interface implemented by the various structures used to compose the 'top-level' per-container data structure...